<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastroEstruturaTemporada</class>
 <widget class="QWidget" name="cadastroEstruturaTemporada">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="2" column="2">
      <widget class="QPushButton" name="cancelar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Fechar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Fases</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_16">
        <item row="0" column="0" colspan="4">
         <widget class="QListWidget" name="fases"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="incluirFase">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Incluir</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="alterarFase">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Alterar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="excluirFase">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Excluir</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="QCheckBox" name="checkBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Fase de Eliminação</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLineEdit" name="nomeFase">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nome:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Rodadas</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="0" column="0" colspan="2">
         <widget class="QListWidget" name="rodadas">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Selecione um Grupo</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="adicionarRodada">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Adicionar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="diminuirRodada">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Diminuir</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Grupos</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="1" column="4">
         <widget class="QPushButton" name="excluirGrupo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Excluir</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="5">
         <widget class="QListWidget" name="grupos">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Selecione uma Fase</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="alterarGrupo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Alterar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QPushButton" name="incluirGrupo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Incluir</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="2">
         <widget class="QLineEdit" name="nomeGrupo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Nome:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
